medical_prescription/tests/test_medical_prescription_order_line.py

```diff
@@ -64,3 +70,13 @@ def test_name_search_none(self):
         self.assertEquals(
             res_len, 6,
         )
+
+    def test_onchange_medical_medication_id(self):
```

Comment:
This test is a false positive. In order to test the onchange, we would want to actually change the `medical_medication_id` attribute on the prescription.

Interestingly though, onchange methods only trigger via UI changes and not ORM. This means that in order to get the test to pass after changing the attribute, you will also need to call the method `record._oncahgne_medical*`